.vscode/settings.json

```diff
@@ -12,5 +12,8 @@
   "typescript.validate.enable": true,
   "javascript.validate.enable": true,
   "editor.formatOnSave": true,
-  "editor.defaultFormatter": "esbenp.prettier-vscode"
+  "editor.defaultFormatter": "esbenp.prettier-vscode",
```

Comment:
Note: this tells VSCode to treat the *.css files as css language files. Somehow, it was thinking they were "tailwind" files 👎